### PR TITLE
KAA-1301: Database configuration - reviewed.

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

**Comment by v-chyzhevskyi** [comment]
LGTM

**Review by v-chyzhevskyi — APPROVED** [review]

**Review by v-chyzhevskyi — APPROVED** [review]

File: doc/Administration-guide/System-Configuration/Database-configuration/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+permalink: /:path/
+sort_idx: 220
+---
+
+{% include variables.md %}
+
+* TOC
+{:toc}
+
+When you complete [Kaa installation]({{root_url}}Administration-guide/System-installation/), the configuration files for each Kaa component will be extracted into the `/etc/kaa-node/` directory.
```
Comment (Polinaa) [inline doc/Administration-guide/System-Configuration/Database-configuration/index.md:13]: /etc/kaa-node/conf/

File: doc/Administration-guide/System-Configuration/Database-configuration/index.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+* `maria-dao.properties.template`, `mariadb-dao.properties.template` -- for MariaDB.
+* `postgre-dao.properties.template`, `postgresql-dao.properties.template` -- for PostgreSQL.
+
+See the table below for configuration parameters contained in the `admin-dao.properties` and `sql-dao.properties` files.
+
+|Parameter name|Default value|Description|
+|--------------|-------------|-----------|
+|`db_name=kaa`||Database name.|
+|`dao_max_wait_tim`|5|Maximum wait time in seconds for DAO history class. Custom property for Kaa **History service**.|
```
Comment (Polinaa) [inline doc/Administration-guide/System-Configuration/Database-configuration/index.md:48]: dao_max_wait_tim**e**

File: doc/Administration-guide/System-Configuration/Database-configuration/index.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+In case of the password or username mismatch, edit the configuration file.
+
+```bash
+$ sudo nano /etc/kaa-node/conf/admin-dao.properties
+
+$ sudo nano /etc/kaa-node/conf/sql-dao.properties
+```
+
+To switch from MariaDB to PostgreSQL, copy the contents of MariaDB configuration files to the Kaa database configuration files.
```
Comment (Polinaa) [inline doc/Administration-guide/System-Configuration/Database-configuration/index.md:118]: copy the contents of **PostgreSQL**

File: doc/Administration-guide/System-Configuration/Database-configuration/index.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+* [MongoDB 2.6.9](http://www.mongodb.org/downloads)
+* [Cassandra 3.5](http://cassandra.apache.org/download/)
+
+By default, MongoDB database is used.
+
+For database installation instructions, see [Third party components]({{root_url}}Administration-guide/System-installation/Single-node-installation/#third-party-components).
+
+The `nosql-dao.properties` template of the configuration property file for SQL database is located in the `/etc/kaa-node/conf/` directory.
+
+The `nosql-dao.properties` file contains the `nosql_db_provider_name` parameter that can be set to **mongodb** (default) or **cassandra** consist of the following parameters:
```
Comment (Polinaa) [inline doc/Administration-guide/System-Configuration/Database-configuration/index.md:140]: remove: *consist of the following parameters:*

File: doc/Administration-guide/System-Configuration/Database-configuration/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Administration-guide/System-Configuration/Database-configuration/index.md:13]: Fixed.

File: doc/Administration-guide/System-Configuration/Database-configuration/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Administration-guide/System-Configuration/Database-configuration/index.md:48]: Fixed.

File: doc/Administration-guide/System-Configuration/Database-configuration/index.md (hunk 3 above)
Comment (hunch7) [reply doc/Administration-guide/System-Configuration/Database-configuration/index.md:118]: Fixed.

File: doc/Administration-guide/System-Configuration/Database-configuration/index.md (hunk 4 above)
Comment (hunch7) [reply doc/Administration-guide/System-Configuration/Database-configuration/index.md:140]: Fixed.

**Pull request merged**